### ISSUE TITLE
Add steps to the slider widget

### ISSUE THREAD

**Issue by GuillaumeDelorme** [UmbraLuminosa/sickle_ui#115]
It would be great to implement a step config for the slider widget, to restrict selectable values to increments of a given size.

HTML input step documentation for reference:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#step

**Comment by eidloi** [comment]
yeah, I had it on my list... *shuffles notebooks*... somewhere :D
